app/src/services/scroll.js

```diff
@@ -183,9 +183,7 @@ class Scroll {
 
         if (this.resultScroll.aggregations) {
             const data = Scroll.serialize(this.resultScroll, this.parsed, this.format);
-            this.stream.write(this.convertDataToDownload(data, this.format, true, false, this.cloneUrl), {
```

Comment:
Everything other than the changed on this file LGTM. I couldn't deterministically find out which implementation of write() this calls - I did find one that matches the signature you implemented, but then again AFAIK this is working as-is, and this change may introduce a bug. I don't want to spend much time validating this, so I'll leave it up to you to confirm this adds no regressions.

Reply:
Here is the travis-ci [build](https://app.travis-ci.com/github/resource-watch/document-adapter/builds/247857062#L559) failing with that syntax. Here is the function [signature](https://nodejs.org/docs/latest-v16.x/api/stream.html#writablewritechunk-encoding-callback) in node 16 that expects a string type for encoding.